roles/openshift_service_catalog/defaults/main.yml

```diff
@@ -1,6 +1,11 @@
 ---
 openshift_service_catalog_remove: false
+
+# Feature Gates
 openshift_service_catalog_async_bindings_enabled: true
+openshift_service_catalog_namespaced_service_brokers_enabled: true
+openshift_service_catalog_catalog_restrictions_enabled: true
```

Comment:
Did you intend for the name to be "...catalog_catalog..."?

Reply:
Yes.  The feature is CatalogRestrictions.